src/vos/tests/vts_aggregate.c

```diff
@@ -203,11 +203,13 @@ phy_recs_nr(struct io_test_args *arg, daos_unit_oid_t oid,
 static int
 lookup_object(struct io_test_args *arg, daos_unit_oid_t oid)
 {
-	struct vos_obj_df	*obj_df = NULL;
+	struct vos_object	*obj = NULL;
 	int			 rc;
+	daos_epoch_range_t	 epr = {0, DAOS_EPOCH_MAX};
 
-	rc = vos_oi_find(vos_hdl2cont(arg->ctx.tc_co_hdl), oid, 1,
-			 DAOS_INTENT_DEFAULT, &obj_df);
+	rc = vos_obj_hold(vos_obj_cache_current(),
+			  vos_hdl2cont(arg->ctx.tc_co_hdl), oid, &epr, true,
+			  DAOS_INTENT_DEFAULT, true, &obj);
```

Comment:
yeah, I suppose it does need that.   I think the issue with find is that another incarnation may potentially exist so we want to guarantee that this incarnation doesn't exist.   I've changed vos_obj_hold so that it doesn't actually keep a reference if the object doesn't exist.   I'll need to look a the code to see if there are cases where we do this that we verify it exists....in such cases, yes, we need to release.